client/state/current-user/selectors.js

```diff
@@ -44,6 +44,21 @@ export function getCurrentUserLocale( state ) {
 }
 
 /**
+ * Returns the isRTL value for the current user.
+ *
+ * @param  {Object} state  Global state tree
+ * @return {?Bool}  Current user is RTL
+ */
+export function getCurrentUserIsRTL( state ) {
```

Comment:
Several comments about the selector naming.

First, when we have an abbreviation, we use camel case for it as well. So `IsRTL` would become `IsRtl`. See https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#naming-conventions for more information and examples.

From the perspective that we're fetching something from the current user object it makes sense to have the selector named like that, but I think it would make more sense for it to be `isCurrentUserRtl` or even the shorter one `isRtl`, because the other one might look odd.

@mtias how would you name that one?